packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCardGrid.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, BoxProps, styled } from '@mui/material'
+import { StyledComponent } from '@emotion/styled'
+
+export const FACET_PLOTS_CARD_TITLE_CONTAINER_CLASSNAME =
+  'FacetPlotsCardTitleContainer'
+
+export const FACET_PLOTS_CARD_PLOT_CONTAINER_CLASSNAME =
+  'FacetPlotsCardPlotContainer'
+
+export const FacetPlotsCardGridContainer: StyledComponent<BoxProps> = styled(
+  Box,
+  {
+    label: 'FacetPlotsCardGridContainer',
+  },
+)(({ theme }) => ({
+  [theme.breakpoints.down('md')]: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(2),
+  },
+  [theme.breakpoints.up('md')]: {
+    /* 
+    The grid is broken up into cards which have subcomponents that should be aligned
+    Ideally we could use CSS Subgrid, but it is not widely supported as of the time of writing
+    See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid
+
+    Instead we will use grid with `display: contents`, and style each section to look like a card.
+     */
```

Comment:
This is super hacky, so I tried to call attention to this as fixable if we ever get `subgrid`

Reply:
How easy would this be to do with `subgrid` support?  If straight-forward, could you provide an example implementation in the comments here?

Reply:
While it would be much easier and less hacky than this implementation, I don't think it would be trivial, so I don't think it would be very valuable to flesh out an implementation now.